packages/opentelemetry-plugin-grpc-js/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/plugin-grpc-js",
+  "private": true,
+  "version": "0.8.3",
+  "description": "OpenTelemetry @grpc/grpc-js automatic instrumentation package.",
```

Comment:
I think description should explain what it is without repeating the name, maybe mention here that this is for older package ?